generate/vim_template/vimrc

```diff
@@ -430,6 +436,10 @@ vnoremap K :m '<-2<CR>gv=gv
 "" Open current line on GitHub
 nnoremap <Leader>o :.Gbrowse<CR>
 
+"" Code folding options
+" You can also do set nofoldenable to prevent folding
+set foldmethod=indent
```

Comment:
I agree with the changes, except this one, as using _fold_ is complex for newcomers to vim.

Reply:
I see and get your point. From my perspective this comes as a essential-setting in vimrc so hence I added that.